src/Main.elm

```diff
@@ -2,55 +2,91 @@ module Main exposing (..)
 
 import Browser
 import Element
+import Element exposing (rgb255)
 import Element.Input
-import Html exposing (Html, button, div, text)
+import Element.Background as Background
+import Html exposing (Html, div, text)
 import Html.Events exposing (onClick)
 
+-- STYLE
 
--- MAIN
+marigold =
+    Element.rgb255 252 186 3
+
+green =
+    Element.rgb255 8 196 27
 
+-- MAIN
 
 main =
   Browser.sandbox { init = init, update = update, view = view }
 
+{-| https://package.elm-lang.org/packages/elm/browser/latest/Browser
+    init, update, and view are defined below |-}
 
 
 -- MODEL
 
-type alias Model = Int
+type alias Counter = Int
```

Comment:
okay I'll change this back! But for a new user, or someone just reading the code not familiar with Elm, it's very weird because the variable "Model" (intuitively) is describing a Counter, and Model better describes something that is for a database, view, or similar. The same for Msg that I will change from Action - in this app it isn't really a message, it's more like an action to be done. I suspect this is one of those "conventions" that I'm not allowed to argue with, that seems totally straight forward to old timers, but programmers like myself are like ??

Reply:
Yes, I can see how that can be confusing. 

One way to refactor this to be clearer might be to use a record instead of an Int. Something like:

```elm
type alias Model =
    { counter : Int }
```

This way you can also start adding other values to the model. Let's imagine you wanted the user to be able to select the text background color - you might change your Model type to something like this:

```elm
type alias Model =
    { counter : Int
    , textBackground : Element.Color
    }
```

Your view function could use it like so:

```elm
            -- snip
            , Element.el [ Background.color model.textBackground ] (Element.text (String.fromInt model.counter))
            -- snip
```

Reply:
I like this much better, because it's more clearly a message to the developer about what Model actually means. Trying this now!